### ISSUE TITLE
Staging Review finding: Long email addresses overflow at high zoom

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#87428]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Sitewide Facilities
**Product:** VBA Regional Offices
**Feature:** Site Modernization

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

At high zoom (400%) or on mobile devices (eg. 320px viewport), long email addresses overflow out of the cards containing them, eg. in the "Get support" and "Get help requesting records" cards.

It's not a huge impact here --- the addresses are mostly visible and pretty legible, and the functionality of the links isn't impacted. But it's still a potential barrier for some users to access that information.

This was flagged by @davidmpickett in the foundational testing ahead of staging review and is being tracked in [va.gov-cms#18367](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18367).

## Recommended action

From @davidmpickett's comments it sounds like there's a possible solution already in the works!

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 1.4.10 Reflow - Level AA](https://www.w3.org/TR/WCAG21/#reflow)
- **Modality:** Browser zoom
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jilladams** [comment]
Says it in ticket body but for ease of re-finding: we are tracking this in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18367

**Comment by Agile6MSkinner** [comment]
Sitewide ticket [#18367](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18367) addressed this and has closed. Closing this collab cycle ticket as well.
